### DISCUSSION TITLE
Download do agente  .msi

### DISCUSSION THREAD

**Discussion by Givannixs** [amidaware/tacticalrmm#1857] [Ideas]
Disponibilizar o agente para download como um arquivo .msi seria extremamente útil. Estou enfrentando desafios significativos ao tentar instalar o agente do RMM através da política de grupo no Active Directory.

Houve alguém que conseguiu instalar o agente através da GPO e poderia me oferecer uma ajuda?

**Comment by dinger1986** [comment]
MSI would be good but what about powershell?

**Reply by Givannixs** [reply]
Nós fizemos tentativas. A questão é que ao executar o script PowerShell para instalar o agente RMM, ele solicita a senha do administrador, já que os usuários são do tipo padrão.

**Comment by silversword411** [comment]
https://docs.tacticalrmm.com/install_agent/#scripting-agent-installation
Click msi.
Follow instructions
